### PR TITLE
Document Launch with AnVIL button

### PR DESCRIPTION
**Pull request by coverbeck**
dockstore/dockstore#3115

Document Launch with AnVIL button. As part of this:

* Now use the same Dockstore screenshot for all the launch with for WDL
* Changed the See Also on the launch pages to only reference other Launch withs,
since there are now a fair number of Launch with's and Azure Batch and AWS Batch seem
less applicable
* Updated copyright
* Fixed build warning

### CODE REVIEW

File: docs/launch-with/anvil-launch-with.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+AnVIL
+=====
+
+Dockstore integrates with the the Analysis, Visualization, and Informatics Lab-space (AnVIL) platform,
```
Comment (denis-yuen) [inline docs/launch-with/anvil-launch-with.rst]: Maybe a link would be helpful
https://www.genome.gov/Funded-Programs-Projects/Computational-Genomics-and-Data-Science-Program/Genomic-Analysis-Visualization-Informatics-Lab-space-AnVIL perhaps

Comment (coverbeck) [reply docs/launch-with/anvil-launch-with.rst]: Added link

**Comment by denis-yuen** [comment]
Pressed wrong button trying to re-review

File: docs/conf.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 # -- Project information -----------------------------------------------------
 
 project = u'Dockstore'
-copyright = u'2019, OICR'
+copyright = u'2020, OICR'
 author = u'Andrew Duncan'
```
Comment (agduncan94) [inline docs/conf.py]: We should update the author.

**Comment by denis-yuen** [comment]
Extra thought, there is also https://docs.dockstore.org/en/develop/end-user-topics/language-support.html

File: docs/conf.py (hunk 2 above)
Comment (coverbeck) [reply docs/conf.py]: To whom?

Comment (coverbeck) [reply docs/conf.py]: Per Slack discussion, changed it to `OICR, UCSC`

**Pull request merged**